R/qualification-ddi.R

```diff
@@ -217,11 +218,43 @@ generateDDIQualificationDDIPlot <- function(data) {
   ddiPlotConfiguration$export$height <- 2 * 1.2 * (data$plotSettings$height / 96)

```

Comment:
or this for instance... We need a centralized way to deal with scaling. This is high prio IMO